zhaquirks/philips/hue7602031P7.py

```diff
@@ -0,0 +1,97 @@
+"""Philips Hue Go 7602031P7 device."""
```

Comment:
We have a new way to define quirks now. I still need to write documentation for it but I think we can replace the majority of this file with:

```python
from zhaquirks.philips import PHILIPS, PhilipsEffectCluster

(
    add_to_registry_v2(
        PHILIPS, "7602031P7"
    )
    .replaces(PhilipsEffectCluster, endpoint_id=11)
)
```